BuildAndTest.cmd

```diff
@@ -64,36 +64,37 @@ if "%ERRORLEVEL%" NEQ "0" (
 goto ExitFailed
 )
 
-msbuild /verbosity:minimal /target:rebuild src\Everything.sln /filelogger /fileloggerparameters:Verbosity=detailed /p:"RunBinSkim=false" /p:"BinSkimVerboseOutput=true"
-set Platform=AnyCPU
+msbuild /verbosity:minimal /target:BuildAndInjectObjectModel src\Sarif\Sarif.csproj /fileloggerparameters:Verbosity=detailed
+if "%ERRORLEVEL%" NEQ "0" (
```

Comment:
Here's where we introduce the separate build step I mentioned in the PR description. We rebuild the SARIF object model from the schema before attempting to compile the `Sarif` project.